### PR TITLE
Order buttons

### PR DESCRIPTION
**Pull request by andrzejkukuryk**
app navigation is now possible

### CODE REVIEW

File: src/components/goToOrdersButton.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  const showOrdersButton = useSelector(showOrdersButtonSelector);
+
+  const handleClickViewOrders = () => {
+    dispatch(updateCurrentScene("ORDER"));
+  };
+
+  return showOrdersButton ? (
+    <AppButton text="view orders" handledFunction={handleClickViewOrders} />
+  ) : (
+    <></>
```
Comment (anna-morawska) [inline src/components/goToOrdersButton.tsx]: more common would be to return null

File: src/components/newOrderButton.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { initialVegetablesState } from "../store/vegetables/vegetablesSlice";
+import { initialEggState } from "../store/egg/eggSlice";
+import { initialSpreadsState } from "../store/spreads/spreadsSlice";
+import { initialServingState } from "../store/serving/servingSlice";
+import { initialToppingState } from "../store/topping/toppingSlice";
+import { initialNameState } from "../store/name/nameSlice";
+
+export function NewOrderButton() {
+  const dispatch = useDispatch();
+  const handleClickNewOrder = () => {
```
Comment (anna-morawska) [inline src/components/newOrderButton.tsx:12]: you can create one action, for example resetState and add extra reducer to each slice which resets the state tothe initial one. This way you want need to create this massive function

https://redux-toolkit.js.org/api/createSlice#extrareducers

File: src/components/goToOrdersButton.tsx (hunk 1 above)
Comment (andrzejkukuryk) [reply src/components/goToOrdersButton.tsx]: Fixed

File: src/components/newOrderButton.tsx (hunk 2 above)
Comment (andrzejkukuryk) [reply src/components/newOrderButton.tsx:12]: Done.

**Pull request merged**